client/admin.go

```diff
@@ -155,36 +155,42 @@ func (c *GrpcAdminClient) auditLogs(ctx context.Context, opts AuditLogOptions) (
 }
 
 func (c *GrpcAdminClient) ListPolicies(ctx context.Context, opts ...ListOpt) ([]*policyv1.Policy, error) {
-	listOptions := &policyListOptions{
-		filters: make([]*requestv1.ListPoliciesRequest_Filter, 0, len(opts)),
-	}
+	listOptions := &policyListOptions{}
 	for _, opt := range opts {
 		opt(listOptions)
 	}
 
-	req := &requestv1.ListPoliciesRequest{
-		Filters: listOptions.filters,
-	}
-
+	req := &requestv1.ListPoliciesRequest{}
 	if listOptions.sortingOptions != nil {
 		order := requestv1.ListPoliciesRequest_SortOptions_ORDER_ASCENDING
 		if listOptions.sortingOptions.descending {
 			order = requestv1.ListPoliciesRequest_SortOptions_ORDER_DESCENDING
 		}
 		req.SortOptions = &requestv1.ListPoliciesRequest_SortOptions{
-			Order:  order,
-			Column: requestv1.ListPoliciesRequest_SortOptions_Column(listOptions.sortingOptions.field),
+			Order: order,
 		}
 	}
 
 	if err := req.Validate(); err != nil {
 		return nil, fmt.Errorf("could not validate list policies request: %w", err)
 	}
 
-	pc, err := c.client.ListPolicies(ctx, req, grpc.PerRPCCredentials(c.creds))
+	p, err := c.client.ListPolicies(ctx, req, grpc.PerRPCCredentials(c.creds))
 	if err != nil {
 		return nil, fmt.Errorf("could not list policies: %w", err)
 	}
 
-	return pc.Policies, nil
+	getReq := &requestv1.GetPolicyRequest{
+		Id: p.PolicyIds,
+	}
+	if err := getReq.Validate(); err != nil {
+		return nil, fmt.Errorf("could not validate get policy request: %w", err)
+	}
+
+	getRes, err := c.client.GetPolicy(ctx, getReq, grpc.PerRPCCredentials(c.creds))
+	if err != nil {
+		return nil, fmt.Errorf("could not get policy: %w", err)
+	}
+
+	return getRes.Policies, nil
```

Comment:
Is this guaranteed to work? If `ListPolicies` returns 100 policies, then `GetPolicy`  would fail because the limit is 25. 
Furthermore, the client should correspond to the actual API methods exposed by the server. If the `ListPolicies` API call only returns the list of policy IDs, then the client's `ListPolicies` method should return a list of policy IDs as well.